label_studio/users/mixins.py

```diff
@@ -4,6 +4,6 @@ def is_annotator(self):
         return False
 
     def has_permission(self, user):
-        if user.active_organization in self.organizations.all():
+        if user.active_organization in self.organizations.filter(organizationmember__deleted_at__isnull=True):
```

Comment:
Same idea here. Should be a query on OrganizationMember that has `active_organization` as a filter instead of `if ... in ....`